### PR TITLE
add parsec-tool

### PR DESCRIPTION
**Pull request by muvarov**
Signed-off-by: Maxim Uvarov <maxim.uvarov@linaro.org>

### CODE REVIEW

**Review by apalos — DISMISSED** [review]

File: meta-ledge-sw/recipes-security/parsec/curl_%.bbappend [hunk 1]
```diff
@@ -17,13 +17,10 @@ EXTRA_OECONF_append_class-target = " \
...
+    cp ${STAGING_ETCDIR_NATIVE}/ssl/certs/ca-certificates.crt ${HOSTTOOLS_DIR}/etc/ssl/certs/ca-certificates.crt
 }
 
-inherit rm_work
-RM_WORK_EXCLUDE_ITEMS += " certs"
-
 EXTRA_OECONF_append_class-native = " \
-    --with-ca-bundle=${WORKDIR}/certs/ca-certificates.crt \
+    --with-ca-bundle=${HOSTTOOLS_DIR}/etc/ssl/certs/ca-certificates.crt \
 "
```
Comment (apalos) [inline meta-ledge-sw/recipes-security/parsec/curl_%.bbappend:26]: Can you add a description on the commit. Why is this needed?

File: meta-ledge-bsp/recipes-security/optee/optee-os_git.bbappend [hunk 2]
```diff
@@ -9,8 +9,6 @@ DEPENDS += "dtc-native"
 DEPENDS += "python3-pyelftools-native dtc-native python3-pycryptodomex-native python3-pycrypto-native"
 
 SRC_URI_append_ledge-qemuarm64 = " file://bc50d971-d4c9-42c4-82cb-343fb7f37896.stripped.elf "
-# random
-SRC_URI_append_ledge-qemuarm64 = " file://b6c53aba-9669-4668-a7f2-205629d00f86.stripped.elf "
```
Comment (apalos) [inline meta-ledge-bsp/recipes-security/optee/optee-os_git.bbappend:13]: What was that UUID used for in the first place?

File: meta-ledge-sw/recipes-tpm2/tpm2-tss/tpm2-tss/ax_pthread.m4 [hunk 3]
```diff
@@ -0,0 +1,332 @@
+# ===========================================================================
```
Comment (apalos) [inline meta-ledge-sw/recipes-tpm2/tpm2-tss/tpm2-tss/ax_pthread.m4:1]: Would it make sense to bump the version on meta-security instead of copying it in here?
and if we do copy it in here we have recipes-4upstream dir, which is specifically for this purpose.
This we need to merge upstream in the end

File: meta-ledge-sw/recipes-security/parsec/files/config.toml [hunk 4]
```diff
@@ -118,10 +118,10 @@ key_info_manager = "on-disk-manager"
...
 # values are "com.intel.tss2.Tabrmd" for "bus_name" and "system" for "bus_type"
-tcti = "device:/dev/tpm0"
+tcti = "tabrmd:bus_name=com.intel.tss2.Tabrmd,bus_type=system"
 # (Required) Authentication value for performing operations on the TPM Owner Hierarchy. The string can
 # be empty, however we strongly suggest that you use a secure passcode.
 # To align with TPM tooling, PARSEC allows "owner_hierarchy_auth" to have a prefix indicating a string value,
 # e.g. "str:password", or to represent a string version of a hex value, e.g. "hex:1a2b3c". If no prefix is
 # provided, the value is considered to be a string.
-owner_hierarchy_auth = "password"
+owner_hierarchy_auth = ""
```
Comment (apalos) [inline meta-ledge-sw/recipes-security/parsec/files/config.toml:127]: Can we squash this in the original patch?

File: meta-ledge-bsp/recipes-security/optee/optee-os_git.bbappend (hunk 2 above)
Comment (muvarov) [reply meta-ledge-bsp/recipes-security/optee/optee-os_git.bbappend:13]: @apalos  ftpm
